packages/server/test/unit/cloud/api/put_protocol_artifact_spec.ts

```diff
@@ -146,37 +170,31 @@ describe('putProtocolArtifact', () => {
     })
 
     describe('and fetch completes successfully', () => {
-      it('resolves', () => {
-        uploadStreamStub.withArgs(
-          mockReadStream,
-          destinationUrl,
-          fileSize, {
-            retryDelay: geometricRetryStub,
-            activityMonitor: mockStreamMonitor,
-          },
-        ).resolves()
+      it('resolves', async () => {
+        putFetchStub.resolves()
 
-        return expect(putProtocolArtifact(filePath, maxFileSize, destinationUrl)).to.be.fulfilled
+        await putProtocolArtifact(filePath, maxFileSize, destinationUrl)
+
+        expect(true).to.be.true
```

Comment:
do we have something to assert here in this test?

Reply:
this is basically the same as chai as promised' expect().to.be.fulfilled - i changed it back to be more clear. I'd unwrapped it for debugging purposes - thanks for the catch! addressed in https://github.com/cypress-io/cypress/pull/29745/commits/401302054063ee68d9a2c0990c6899f1b82f295f